feature/danggn/src/main/java/com/mashup/feature/danggn/ranking/DanggnRankingViewModel.kt

```diff
@@ -0,0 +1,43 @@
+package com.mashup.feature.danggn.ranking
+
+import com.mashup.core.common.base.BaseViewModel
+import com.mashup.feature.danggn.data.dto.DanggnMemberRankResponse
+import com.mashup.feature.danggn.data.repository.DanggnRepository
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+import javax.inject.Inject
+
+@HiltViewModel
+class DanggnRankingViewModel @Inject constructor(
+    private val danggnRepository: DanggnRepository
+) : BaseViewModel() {
+    companion object {
+        private const val GENERATION_NUMBER = 13
+    }
+
+    private val _mashUpRankingList: MutableStateFlow<List<DanggnMemberRankResponse>> =
+        MutableStateFlow(
+            listOf()
+        )
+    val mashUpRankingList = _mashUpRankingList.asStateFlow()
+
+    init {
+        mashUpScope {
+            updateAllRankingList()
+        }
+    }
+
+    /**
+     * 밖에서도 호출할 수 있도록 private가 아닌 internal로 만들었습니다.
+     */
+    internal suspend fun updateAllRankingList() {
+        val allMemberRankingResult = danggnRepository.getAllDanggnRank(GENERATION_NUMBER)
+        if (allMemberRankingResult.isSuccess()) {
+            val rankingList = danggnRepository.getAllDanggnRank(GENERATION_NUMBER)
+                .data?.allMemberRankList ?: listOf()
```

Comment:
```suggestion
                .data?.allMemberRankList ?: emptyList()
```
ㅋㅋㅋㅋ 사소할 수 있는데 default 값으로 비어있는 리스트로 넣고 싶을 때 `emptyList`를 사용해도 좋을 것 같네용

Reply:
ㅋㅋㅋㅋㅋ이거 emptyList 했다가... 바꾼건뎅 목데이터가 import 가 잘못돼서 List<Any> 타입으로 나와버려서 바꿈...요...
여튼 감사링~